<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="produto">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>660</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 25pt &quot;Courier&quot;;
background-color: rgb(253, 255, 216);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="empresa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;

background-color: rgb(223, 213, 255);

border-color: rgb(162, 162, 162);

border-style: Outset;
border-width: 2px;
border-radius: 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Users/Helio/.designer/backup/CADASTRO.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_fecha">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>610</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;F10&gt;FECHAMENTO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Cliente:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Condicao de Pagamento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de Dias:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Observacao:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Media de Dias:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_cliente">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>520</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Vendedor:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_sair">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>610</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Codigo Condicao Pagamento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero do Pedido:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero da Ordem Servico:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Recuperar Pedido</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Recuperar O.S.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>520</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_cliente_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>520</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_cliente_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>520</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Codigo do Representante:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Codigo do Faturamento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de Pedido:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>cb_cliente</tabstop>
  <tabstop>bt_fecha</tabstop>
  <tabstop>bt_sair</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
